docs/low-code-spark/extensibility/gem-builder.md

```diff
@@ -6,20 +6,857 @@ description: Gem-Builder
 tags: []
 ---
 
-Prophecy Data Engineering system can be extended with **Gem Builder** to add **your own standard sources, targets and
-transforms** to Spark and roll it out to your entire team. You can build **your own frameworks with** custom Data Quality
-library, or auditing library.
+:::caution Enterprise Only
+
+Please [contact us](https://www.prophecy.io/request-a-demo) to learn more about the Enterprise offering.
+
+:::
+
+[Gems](docs/concepts/gems.md) are operations you can add to your Pipeline to perform actions on your data. The **Gem Builder** allows you to create custom UI experiences that will generate optimized code wherever it's used. You can add your own custom sources, targets, and transformations and roll them out to your entire team. You can even build a custom Data Quality library, or an auditing library.
 
 <div class="video-container">
 <iframe src="https://www.youtube.com/embed/K23pOatAeVE" title="YouTube video player" frameborder="0"
 allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
```

Comment:
This video should be re-created with the new UI. 
The same audio could be used :-)

Reply:
I will do that in a separate PR may be in few days. :-)